test/benchmark/Compare.cpp

```diff
@@ -27,8 +38,12 @@ static void BM_Throughput_libhat(benchmark::State& state) {
     const auto end = std::to_address(buf.end());
 
     const auto sig = hat::parse_signature(test_pattern).value();
+    // Pre-bake the scanner so it's crispy fast
+    hat::scanner scanner(sig);
+    
     for (auto _ : state) {
-        benchmark::DoNotOptimize(hat::find_pattern(begin, end, sig));
+        // Use the pre-compiled scanner
+        benchmark::DoNotOptimize(scanner.find(std::execution::par, begin, end));
```

Comment:
A multi-threaded benchmark should be defined separately. The purpose of the Compare benchmark is to show single-threaded throughput.